sw/device/silicon_creator/manuf/provisioning_inputs.bzl

```diff
@@ -25,14 +25,15 @@ FT_PERSONALIZE_KEYS = [
     "//sw/device/silicon_creator/manuf/keys/fake:rma_unlock_token_export_key.sk_hsm.der",
 ]
 
+# Note that uds-auth-key-id below is the actual hash of the public key of cert_endorsement_key.sk.der
 FT_PROVISIONING_INPUTS = _DEVICE_ID_AND_TEST_TOKENS + """
   --target-mission-mode-lc-state="prod"
   --host-ecc-sk="$(rootpath //sw/device/silicon_creator/manuf/keys/fake:rma_unlock_token_export_key.sk_hsm.der)"
   --cert-endorsement-ecc-sk="$(rootpath //sw/device/silicon_creator/manuf/keys/fake:cert_endorsement_key.sk.der)"
   --rom-ext-measurement="0x11111111_11111111_11111111_11111111_11111111_11111111_11111111_11111111"
   --owner-manifest-measurement="0x22222222_22222222_22222222_22222222_22222222_22222222_22222222_22222222"
   --owner-measurement="0x33333333_33333333_33333333_33333333_33333333_33333333_33333333_33333333"
-  --uds-auth-key-id="0x11223344_55667788_99112233_44556677_88991122"
+  --uds-auth-key-id="0xfe584ae7_53790cfd_8601a312_fb32d3c1_b822d112"
```

Comment:
just an observation: we're going to have to change this again and again each time CA cert changes, maybe CA cert should be passed as a command line input?

Reply:
Well this is really just for running this flow in CI, it is already a command line input to the FT provisioning binary which can be run with `bazel run ...`. We also have script that runs both FT and CP.